doc/docs/modules/ROOT/pages/backup.adoc

```diff
@@ -50,34 +50,76 @@ In Neo4j 4.0, the system can be multidatabase; most systems have at least 2 DBs,
 
 First you want to create a kubernetes secret that contains the content of your account service key.  This key must have permissions to access the bucket and backup set that you're trying to restore. 
 
+**AWS**
+
+- You must create the credential file and this file should look like this:
+```aws-credentials
+[default]
+region=
+aws_access_key_id=
+aws_secret_access_key=
+```
+
+- You have to create a secret for this file
+```shell
+kubectl create secret generic neo4j-aws-credentials \
+    --from-file=credentials=aws-credentials
```

Comment:
Add note:   the `credentials=` part is critical.  When attempting to test your PR I messed this up twice, not reading carefully enough.  The internal container expects of course to have a file named "credentials" mounted.  If you instead pass `--from-file=foobar.txt` even though you have the right data, the process breaks, because the container doesn't find the file `credentials` as it expects. Something to note here because it's a very tiny detail but I tripped over it.